src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ComponentResourceManager.cs

```diff
@@ -51,6 +52,7 @@ private CultureInfo NeutralResourcesCulture
         /// to the corresponding property on the object. If there is no matching
         /// property the resource will be ignored.
         /// </summary>
+        [RequiresUnreferencedCode("The Type of value cannot be statically discovered.")]
```

Comment:
NIT: Share the string of both of these API's attributes?

Reply:
In some cases it helps sharing strings (like the AttributeCollection case). For these strings, I found that sometimes the method uses the word `value`, sometimes `instance`, and sometimes, `component`. So I just started copying the strings because I didn't want to reference the wrong word.